apis/sgv2-restapi/src/main/java/io/stargate/sgv2/restapi/service/resources/HealthResource.java

```diff
@@ -22,13 +22,20 @@
 import jakarta.ws.rs.core.MediaType;
 import jakarta.ws.rs.core.Response;
 
+/** Legacy Health endpoint used by some routers to check if the service is up and running. */
 @Path("/")
-@Produces(MediaType.APPLICATION_JSON)
+@Produces(MediaType.TEXT_PLAIN)
 @Singleton
 public class HealthResource {
   @GET
-  @Path("/health")
+  @Path("health")
   public Response health() {
     return Response.status(Response.Status.OK).entity("UP").build();
   }
+
+  @GET
```

Comment:
Adding at `/ping` because Quarkus reserves `/` for contents of `index.html`; was not able to find a way to override dynamically.
Not 100% sure we will be using this endpoint but at least it is defined should we want to do that.